### PR TITLE
first improvements to docs.

### PR DESCRIPTION
**Pull request by 8ball030**
## Proposed changes

Simplify the intro, 

Remove a lot of unnecessary wording,

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -99,13 +94,11 @@ you can follow the rest of the guide the same way as if not using docker.
...
 </details>
 
-## Preliminaries
+## Getting Started
 
 Ensure, you are in a clean working directory:
 
-- either you create it manually `mkdir my_aea_projects/ && cd my_aea_projects/`, then add an empty directory called `packages` with the following command `mkdir packages/`,
-
-- or you clone the template repo as described in `Approach 1` in the <a href="../development-setup#approach-1">development setup</a> guide.
```
Comment (DavidMinarsch) [inline docs/quickstart.md:108]: We could create a template repo too

File: docs/quickstart.md [hunk 2]
```diff
@@ -169,85 +162,26 @@ Confirm password:
...
 ```
 
 <div class="admonition note">
   <p class="admonition-title">Note</p>
   <p>If you would rather not create an account on the registry at this point, then run <code>aea init --local</code> instead.</p>
 </div>
 
+
+
 ## Echo skill demo
```
Comment (DavidMinarsch) [inline docs/quickstart.md:175]: For now let's ask the user to copy the packages folder from the repo via `svn export` so they can use `--local`. We don't have a remote registry atm.

File: docs/quickstart.md [hunk 3]
```diff
@@ -169,85 +162,26 @@ Confirm password:
...
 
 This is a simple demo that introduces you to the main components of an AEA. 
 
 The fastest way to have your first AEA is to fetch one that already exists!
 
 ``` bash
-aea fetch fetchai/my_first_aea:0.27.0
```
Comment (DavidMinarsch) [inline docs/quickstart.md:187]: Note, we no longer have the `aea interact` command in the `open-aea`. So the user has to use `echo` to file instead

File: docs/quickstart.md (hunk 2 above)
Comment (8ball030) [reply docs/quickstart.md:141]: Is this needed? I ran this locally and didnt have any issues. I can confirm on a new machine, and the command to svn pull is;
```
svn checkout https://github.com/valory-xyz/open-aea/trunk/packages packages```

**Review by DavidMinarsch — APPROVED** [review]
LGTM

File: docs/demos/echo_demo.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+This is a simple demo that introduces you to the main components of an AEA. 
+
+A full break down of the development is covered within the <a href="../../quickstart/">Development Quickstart</a>. It is highly recommended that developers begin by following the quick start!
+
+The fastest way to have your first AEA is to fetch one that already exists!
+
+``` bash
+aea fetch fetchai/my_first_aea:0.27.0 --remote
+cd my_first_aea
```
Comment (DavidMinarsch) [inline docs/demos/echo_demo.md:9]: Please use the local approach! They should clone the packages folder!

File: docs/demos/echo_demo.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+aea install
+```
+
+### Add and create a private key
+
+All AEAs need a private key to run. Add one now:
+
+``` bash
+aea generate-key fetchai
+aea add-key fetchai
```
Comment (DavidMinarsch) [inline docs/demos/echo_demo.md:23]: Please use ethereum approach

File: docs/quickstart.md (hunk 2 above)
Comment (DavidMinarsch) [reply docs/quickstart.md:141]: Yes!

File: docs/quickstart.md [hunk 6]
```diff
@@ -156,8 +149,8 @@ You should see a similar output (with your input instead of the sample username
 ``` bash
 Do you have a Registry account? [y/N]: n
 Create a new account on the Registry now:
-Username: fetchai
-Email: hello@fetch.ai
+Username: valory
+Email: hello@valory.xyz
 Password:
 Please make sure that passwords are equal.
 Confirm password:
```
[the remark this answers is not in the document]
Comment (8ball030) [reply docs/quickstart.md]: I have added some scaffold docs based on hist commits

**Pull request merged**